css/styles.css

```diff
@@ -0,0 +1,602 @@
+  font-size: 1.1rem;
+}
+
+.card-text {
+  color: #fff;
+  font-weight: 700;
+  font-family: inherit;
+}
+
+.team {
+  padding: 3rem 4rem;
+}
+
+.program-heading.group {
+  color: #272a31;
+  background-color: transparent;
+}
+
+.team-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 15px;
+}
+
+.frame img {
+  width: 10rem;
+  height: 11rem;
+  object-fit: cover;
+}
+
+.individual {
+  display: flex;
+  gap: 15px;
+}
+
+.team-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.name {
+  color: #272a31;
+  font-weight: 700;
+  font-size: 1.5rem;
+}
+
+.role {
+  font-style: italic;
+  color: #e61f09;
+  font-size: 1rem;
+}
+
+.line {
+  width: 2.5rem;
+  height: 0.1rem;
+  background-color: #a2a8b5;
+  margin: 0.5rem 0;
+}
+
+.partners {
+  background: #272a31;
+  padding: 5rem 0;
+  margin-top: 5rem;
+}
+
+.partners-header {
+  font-size: 1.8rem;
+  color: #fff;
+  text-align: center;
+  position: relative;
+}
+
+.partners-header::after {
+  content: "";
+  height: 3px;
+  width: 15%;
+  background: linear-gradient(to right, #d3d3d3, #f24432);
+  position: absolute;
+  margin: auto;
+  bottom: -5px;
+  left: 0;
+  right: 0;
+}
+
+.partner-list {
+  margin-top: 30px;
+  display: flex;
+  gap: 15px;
+  align-items: center;
+  justify-content: center;
+}
+
+.partner-img {
+  width: 80px;
+  height: 25px;
+}
+
+.hero-about {
+  background-image: url(../images/bg-3.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-color: #d3d3d3;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+  padding: 3rem 4rem;
+  font-family: "Lato", sans-serif;
+}
+
+.about-text {
+  text-align: center;
+  font-size: 1.5rem;
+  color: #ed5342;
+  font-weight: bold;
+  margin-top: 50px;
+}
+
+.detail {
+  text-align: center;
+  background-color: #fff;
+  padding: 1rem 2rem;
+}
+
+.about-detail {
+  text-align: center;
+  font-size: 2.5rem;
+  color: #ed5342;
+  font-weight: bold;
+}
+
+.about-logo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 5rem;
+  gap: 25px;
+  padding: 3rem 4rem;
+}
+
+.logo-head {
+  position: relative;
+  font-size: 1.2rem;
+  margin-top: 30px;
+}
+
+.logo-head::after {
+  content: "";
+  height: 3px;
+  width: 15%;
+  background: linear-gradient(to right, #d3d3d3, #f24432);
+  position: absolute;
+  margin: auto;
+  bottom: -5px;
+  left: 0;
+  right: 0;
+}
+
+.logo-detail {
+  text-align: center;
+}
+
+.logo-design {
+  border: 1px solid #d3d3d3;
+  width: 200px;
+  height: auto;
+}
+
+.workshop {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  gap: 20px;
+  margin-top: 2rem;
+}
+
+.workshop-1 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-1.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
```

Comment:
- Hey, please, let's increase the height to 12rem so our design is close to matching the design template.